.changes/unreleased/Improvements-726.yaml

```
@@ -0,0 +1,6 @@
+component: runtime
+kind: Improvements
+body: Bumped pulumi to master, bumped go.mod for pulumi, excluded failing tests
```

```suggestion
body: Bump pulumi to master, bump go.mod for pulumi, exclud failing tests
```

But really this changelog isn't useful to users at all, so I think we're better off removing it.